docs: KFilterGroup documentation [KHCP-19484] (#3098)
Kong/kongponents: docs/components/filter-group.md

# Filter Group

> KFilterGroup is a component that displays provided filters and allows a user to set the values of those filters.

Reviewer: ```suggestion
KFilterGroup is a component that provides an interface for displaying and applying filter values.
```

    }"
  >
    <template #created-content>
      <MyCustomContent />
    </template>
  </KFilterGroup>
```

### selectorLabel

Sets the text content in the filter selector.

<ClientOnly>
  <KFilterGroup
    v-model="selectorLabelSelection"
    :filters="selectorLabelFilters"
    selector-label="Custom filter selector"
  />
</ClientOnly>

```html
  <KFilterGroup
    v-model="selection"
    :filters="filters"
    selector-label="Custom filter selector"
  />
```

## Slots

### Filter Content

You can slot your custom content into each filter. Use the corresponding filter key from the [`filters` prop](#filters) to target the appropriate `*-content` slot. When you do this, the KFilterGroup is no longer able to determine what the user's selection is for that filter therefore you must manage the `selection.*` for that filter.

<ClientOnly>
  <KFilterGroup
    v-model="customNameSelection"
    :filters="customNameFilters"
    @apply="onCustomApply"
    @clear="onCustomClear"
    @open="onCustomOpen"
  >
    <template #customName-content>
      <div style="width: 300px; transform: rotate(5deg); padding: 10px 5px;">
        <KInput
          v-model="customName"
          label="Value"

>           placeholder="Your name, but rotated 5 degrees"

Author: I wanted to do the KDateTimePicker, but the code for it is shockingly verbose. KSlider might be better, I'll take a look.
